addon-test-support/@ember/test-helpers/settled.js

```diff
@@ -107,17 +107,28 @@ export function isSettled(options) {
   return true;
 }
 
+const TIMEOUTS = [0, 1, 2, 5];
+const MAX_TIMEOUT = 10;
+
 export default function settled(options) {
   return new EmberPromise(function(resolve) {
-    let watcher = global.setInterval(function() {
-      let settled = isSettled(options);
-      if (settled) {
-        // Stop polling
-        global.clearInterval(watcher);
-
-        // Synchronously resolve the promise
-        run(null, resolve);
+    function scheduleCheck(counter) {
+      let timeout = TIMEOUTS[counter];
+      if (timeout === undefined) {
+        timeout = MAX_TIMEOUT;
       }
-    }, 10);
+
+      global.setTimeout(function() {
```

Comment:
FWIW, I _think_ this is related to your `timecop` issue.  I'll fix to use `nextTick` (which is captured eagerly much earlier) once #258 lands.

Reply:
except, it's not: https://github.com/emberjs/ember-test-helpers/issues/245#issuecomment-351068278 🤔

Reply:
should still be changed though, I agree